spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala

```diff
@@ -40,7 +40,7 @@ class DnnGraph(
   private val _variables: Option[(Array[Tensor[Float]], Array[Tensor[Float]])] = None,
   private val enableExcludeChecking: Boolean = true)
   extends Graph[Float](_inputs, _outputs, _variables)
-  with MklDnnLayer {
+  with MklDnnLayer with MklInt8Convertible {
```

Comment:
When will you use scales in dnn graph

Reply:
For test. Same as the `Graph`. I need to make sure the `DnnGraph` can worked fine first.